themes/default/content/registry/packages/aws-apigateway/how-to-guides/aws-apigateway-go-routes.md

```diff
@@ -12,12 +12,12 @@ layout: package
 <!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
 <!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/mktutorial. -->
 
-<p class="mb-4 flex">
-    <a class="flex flex-wrap items-center rounded-md font-display text-lg text-white bg-blue-600 border-2 border-blue-600 px-2 mr-2 whitespace-no-wrap hover:text-white" style="height: 45px;" href="https://github.com/pulumi/examples/tree/master/aws-apigateway-go-routes" target="_blank">
-        <span><i class="fab fa-github pr-2"></i> View Code</span>
+<p class="mb-4 inline-flex items-center">
+    <a class="inline-flex flex-wrap items-center rounded-md font-display text-lg text-white bg-blue-600 border-2 border-blue-600 px-3 mr-2 whitespace-no-wrap hover:text-white" style="height: 45px; line-height: 40px;" href="https://github.com/pulumi/examples/tree/cnunciato/button-fixes/aws-apigateway-go-routes" target="_blank">
```

Comment:
not sure if I completely understand why these are pointed to your branch

Reply:
Ah yes, this was just so I could preview the changes before https://github.com/pulumi/examples/pull/1640 merged.